prettier/vendor/standalone.js

```diff
@@ -26228,6 +26335,130 @@ function isSimpleFlowType(node) {
+  return parentNode.type === "TaggedTemplateExpression" && parentNode.quasi === node && parentNode.tag.type === "MemberExpression" && parentNode.tag.property.type === "Identifier" && parentNode.tag.property.name === "each" && (parentNode.tag.object.type === "Identifier" && jestEachTriggerRegex.test(parentNode.tag.object.name) || parentNode.tag.object.type === "MemberExpression" && parentNode.tag.object.property.type === "Identifier" && (parentNode.tag.object.property.name === "only" || parentNode.tag.object.property.name === "skip") && parentNode.tag.object.object.type === "Identifier" && jestEachTriggerRegex.test(parentNode.tag.object.object.name));
+}
+
+function printJestEachTemplateLiteral(node, expressions, options) {
+  /**
+   * a    | b    | expected
+   * ${1} | ${1} | ${2}
+   * ${1} | ${2} | ${3}
+   * ${2} | ${1} | ${3}
+   */
+  var headerNames = node.quasis[0].value.raw.trim().split(/\s*\|\s*/);
+
+  if (headerNames.length > 1 || headerNames.some(function (headerName) {
+    return headerName.length !== 0;
+  })) {
+    var parts = [];
+    var stringifiedExpressions = expressions.map(function (doc$$2) {
+      return "${" + printDocToString$1(doc$$2, Object.assign({}, options, {
+        printWidth: Infinity,
+        endOfLine: "lf"
+      })).formatted + "}";
+    });
+    var tableBody = [{
+      hasLineBreak: false,
+      cells: []
+    }];
+
+    for (var i = 1; i < node.quasis.length; i++) {
+      var row = tableBody[tableBody.length - 1];
+      var correspondingExpression = stringifiedExpressions[i - 1];
+      row.cells.push(correspondingExpression);
+
+      if (correspondingExpression.indexOf("\n") !== -1) {
+        row.hasLineBreak = true;
+      }
+
+      if (node.quasis[i].value.raw.indexOf("\n") !== -1) {
+        tableBody.push({
+          hasLineBreak: false,
+          cells: []
+        });
+      }
+    }
+
+    var maxColumnCount = tableBody.reduce(function (maxColumnCount, row) {
+      return Math.max(maxColumnCount, row.cells.length);
+    }, headerNames.length);
+    var maxColumnWidths = Array.from(new Array(maxColumnCount), function () {
+      return 0;
+    });
+    var table = [{
+      cells: headerNames
+    }].concat(tableBody.filter(function (row) {
+      return row.cells.length !== 0;
+    }));
+    table.filter(function (row) {
+      return !row.hasLineBreak;
+    }).forEach(function (row) {
+      row.cells.forEach(function (cell, index) {
+        maxColumnWidths[index] = Math.max(maxColumnWidths[index], getStringWidth$2(cell));
+      });
+    });
+    parts.push("`", indent$6(concat$11([hardline$8, join$7(hardline$8, table.map(function (row) {
+      return join$7(" | ", row.cells.map(function (cell, index) {
+        return row.hasLineBreak ? cell : cell + " ".repeat(maxColumnWidths[index] - getStringWidth$2(cell));
+      }));
+    }))])), hardline$8, "`");
+    return concat$11(parts);
+  }
+}
+/** @param node {import("estree").TemplateLiteral} */
```

Comment:
Could that error be caused by this line?

Reply:
Resolved, Looks like this is the problem.

Thanks